docs/git-comparison.md

```diff
@@ -222,7 +222,7 @@ parent.
     </tr>
     <tr>
       <td>Temporarily put away the current change</td>
-      <td>Not needed</td>
+      <td>Automatically preserved in the working-copy commit. You can always check out a different commit.</td>
```

Comment:
Perhaps we should explicitly suggest `jj new @-`?

Reply:
True, `jj new @-` is a good equivalent of `git stash`. See how this looks. I don't know if we should mention that the user never needs to stash before rebasing etc.

Reply:
I think this is resolved now, the results looks very nice to me.

Reply:
Thanks for reviewing.

Reply:
I just checked out jj to make this change, only to realize it's already in the repo, but just not in the website yet ! I can confirm that `jj new @-` would have been the thing I want to see (for the `git stash && check something && git stash pop` kind of use case).

Reply:
Regarding the website, there's a version selector at the top. The update is in the prerelease version: https://martinvonz.github.io/jj/prerelease/git-comparison/